src/annotator/anchoring/html.ts

```diff
@@ -63,50 +66,45 @@ export function anchor(root, selectors, options = {}) {
 
   // From a default of failure, we build up catch clauses to try selectors in
   // order, from simple to complex.
-  /** @type {Promise<Range>} */
-  let promise = Promise.reject('unable to anchor');
+  let promise: Promise<Range> = Promise.reject('unable to anchor');
 
   if (range) {
     // Const binding assures TS that it won't be re-assigned when callback runs.
     const range_ = range;
     promise = promise.catch(() => {
-      let anchor = RangeAnchor.fromSelector(root, range_);
+      const anchor = RangeAnchor.fromSelector(root, range_);
       return querySelector(anchor, options).then(maybeAssertQuote);
     });
   }
 
   if (position) {
     const position_ = position;
     promise = promise.catch(() => {
-      let anchor = TextPositionAnchor.fromSelector(root, position_);
+      const anchor = TextPositionAnchor.fromSelector(root, position_);
       return querySelector(anchor, options).then(maybeAssertQuote);
     });
   }
 
   if (quote) {
     const quote_ = quote;
     promise = promise.catch(() => {
-      let anchor = TextQuoteAnchor.fromSelector(root, quote_);
+      const anchor = TextQuoteAnchor.fromSelector(root, quote_);
       return querySelector(anchor, options);
     });
   }
 
   return promise;
 }
 
-/**
- * @param {Element} root
- * @param {Range} range
- */
-export function describe(root, range) {
+export function describe(root: Element, range: Range) {
   const types = [RangeAnchor, TextPositionAnchor, TextQuoteAnchor];
   const result = [];
-  for (let type of types) {
+  for (const type of types) {
     try {
       const anchor = type.fromRange(root, range);
       result.push(anchor.toSelector());
     } catch (error) {
-      continue;
+      // If resolving some anchor fails, we want to just skip it silently
```

Comment:
This `continue` statement does nothing, as it's the last statement in the loop. I have replaced it with a comment.